docs/proposals-accepted/202107-protobuf-openapi-httpapi.md

```diff
@@ -0,0 +1,73 @@
+| type     | title                                                        | status   | menu               |

```

Comment:
I think we use different format of front matter - hopefully we can get rid of this too, but right now it's required to be yaml between `---` letters.